lib/web_ui/dev/macos_info.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'dart:convert';
+
+import 'utils.dart';
+
+class MacOSInfo {
+  Future<void> collectInformation() async {
```

Comment:
Suggestion: the name `collectInformation` sounds like it'll return the information, not print it. What about `printInformation`?

Reply:
done.